tools/fleetctl-docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:stable-slim
+
+RUN apt-get update \
+  && dpkg --add-architecture i386 \
+  && apt update \
+  && apt install -y --no-install-recommends ca-certificates cpio libxml2 wine wine32 libgtk-3-0 \
+  && rm -rf /var/lib/apt/lists/* 
+
+# copy macOS dependencies
+COPY --from=fleetdm/bomutils:latest /usr/bin/mkbom /usr/local/bin/xar /usr/bin/
+COPY --from=fleetdm/bomutils:latest /usr/local/lib /usr/local/lib/
```

Comment:
midway implementing this I found out about `COPY --from`, which enables us to separate concerns nicely and avoid copying and pasting a LOT of code (`fleetdm/bomutils` and `fleetdm/wix` both use helper scripts, not only their Dockerfiles)

@lucasmrod what do you think? This would mean we won't deprecate these two images, but instead use them to build this one, which is different from what we spoke.

I'm not set on this, so happy to change it back to what I had before.

Reply:
+2: +1 to reuse what already works, and +1 for separation of concerns.